charts/super-agent-deployment/templates/_helpers.tpl

```diff
@@ -0,0 +1,23 @@
+{{- /*
+`newrelic-super-agent.config` builds the configuration from config on the values and add more config options like
+cluster name, licenses, and custom attributes
+*/ -}}
+{{- define "newrelic-super-agent.config" -}}
+{{- /*
+TODO:
+    * `licenseKey`: comes from the common library but it would not be needed as it is an ingestion API Key, not a REST one.
+    * `rbac`: the is a placeholder for RBAC that simply list pods. this has to be narrowed to the use case of this agent.
```

Comment:
this was implemented already, right?

Reply:
I removed that comment as every TODO is not commented on the `values.yaml` of the deployment template.

Removed on [#cef489e](https://github.com/newrelic/helm-charts/pull/1155/commits/cef489ec5fdd577cc4fc46010ee5d57b1e368705)